client/webserver/site/src/js/charts.ts

```diff
@@ -1437,17 +1437,6 @@ function clamp (v: number, min: number, max: number): number {
   return v
 }
 
-/* labelSpecs is specifications for axis tick labels. */
-const labelSpecs = {
-  minimumSignificantDigits: 4,
-  maximumSignificantDigits: 5
-}
-
-/* formatLabelValue formats the provided value using the labelSpecs format. */
-function formatLabelValue (x: number) {
-  return x.toLocaleString('en-us', labelSpecs)
-}
```

Comment:
I didn't quite get the idea behind formatting those values this way, but the way I'm formatting now in this PR seems reasonable from what testing I've done.